indicators/Kama/Kama.cs

```diff
@@ -60,19 +60,22 @@ public static partial class Indicator
                         // kama calculation
                         decimal? pk = results[i - 1].Kama;  // prior KAMA
                         r.Kama = pk + sc * sc * (h.Close - pk);
+                        r.ER = er;
                     }
 
                     // handle flatline case
                     else
                     {
                         r.Kama = h.Close;
+                        r.ER = 0;
                     }
                 }
 
                 // initial value
                 else if (index == erPeriod)
                 {
                     r.Kama = h.Close;
+                    r.ER = 0;
```

Comment:
I think we should leave this as `null` or unset here instead of `0`.  A zero value would imply the last X periods were an extreme flatline case.